crates/bevy_scene/src/serde.rs

```diff
@@ -1,9 +1,11 @@
 use crate::{DynamicEntity, DynamicScene};
```

Comment:
Seems like it might be better to exclude this whole module, rather than annotating each code block inside of it.

Reply:
I made it so all elements are disabled from within the ``serde`` module.

Would you like it if the imports of the ``serde`` module would be disabled instead?

Reply:
Yeah typically we add the `#[cfg(feature =` line over the `mod serde`. This way the module is actually non-existent when the feature is disabled.